app/(afterLogin)/(timesheetInformation)/timesheet/dashboard/_components/personal/MyAttendanceReport.tsx

```
@@ -1,71 +1,126 @@
+            defaultValue="All"
+            className="w-32  h-12"
+            options={statusOptions}
+            onChange={(value) => setStatusOnAttendance(value)}
+          />
 
-          <RangePicker className="w-32  h-12" />
+          <RangePicker
+            className="w-32  h-12"
+            allowClear
+            onChange={(value) => {
+              if (value && value[0] && value[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)